module/surveys/helper.js

```diff
@@ -874,13 +878,23 @@ module.exports = class SurveysHelper {
 
         let solutionDocument = await solutionsHelper.solutionDocuments(
           solutionQueryObject,
-          solutionDocumentProjectionFields
+          [...solutionDocumentProjectionFields, "referenceQuestionSetId", "type"],
         );
 
         if (!solutionDocument.length) {
           throw new Error(messageConstants.apiResponses.SOLUTION_NOT_FOUND);
         }
 
+        let referenceQuestionSetId = solutionDocument[0]?.referenceQuestionSetId;
```

Comment:
@jayesh9747  why this line is repeated solutionDocument[0]?.referenceQuestionSetId;